explorer/ast/expression.cpp

```diff
@@ -26,6 +26,9 @@ auto IntrinsicExpression::FindIntrinsic(std::string_view name,
   if (name == "Print") {
     return Intrinsic::Print;
   }
+  if(name == "Rand"){
```

Comment:
IMO this name to general. how if we add more semantic word since random word can interpreted as random number,  random uuid, random string , etc ?
wdyt ?

Reply:
@mico258 you are reight, in the case that Print and Rand stay builtin functions. I don't think so, in the future Rand should be a api call.